hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServer.java

```diff
@@ -45,7 +46,7 @@ SCMRatisResponse submitRequest(SCMRatisRequest request)
   /**
    * Returns roles of ratis peers.
    */
-  List<String> getRatisRoles();
+  List<String> getRatisRoles() throws UnknownHostException;
```

Comment:
nit. Good to through "IOException" here, rather than an implementation detail like UnknownHostException.

Reply:
Thanks for the great input, code was updated.